routers/web/web.go

```diff
@@ -1210,13 +1210,15 @@ func RegisterRoutes(m *web.Route) {
 				m.PostOptions("/git-receive-pack", repo.ServiceReceivePack)
 				m.GetOptions("/info/refs", repo.GetInfoRefs)
 				m.GetOptions("/HEAD", repo.GetTextFile("HEAD"))
+				m.GetOptions("/config", repo.GetTextFile("config"))  // needed by git-annex's dumb http mode
 				m.GetOptions("/objects/info/alternates", repo.GetTextFile("objects/info/alternates"))
 				m.GetOptions("/objects/info/http-alternates", repo.GetTextFile("objects/info/http-alternates"))
 				m.GetOptions("/objects/info/packs", repo.GetInfoPacks)
 				m.GetOptions("/objects/info/{file:[^/]*}", repo.GetTextFile(""))
 				m.GetOptions("/objects/{head:[0-9a-f]{2}}/{hash:[0-9a-f]{38}}", repo.GetLooseObject)
 				m.GetOptions("/objects/pack/pack-{file:[0-9a-f]{40}}.pack", repo.GetPackFile)
 				m.GetOptions("/objects/pack/pack-{file:[0-9a-f]{40}}.idx", repo.GetIdxFile)
+				m.GetOptions("/annex/objects/{hash1}/{hash2}/{keyDir}/{key}", repo.GetAnnexObject) // for git-annex
```

Comment:
Because I cooked some delicious copy pasta.

Reply:
(I agree I'll fix)

Reply:
Actually, after stewing this for months, I'd rather conform to their convention. There's no harm in letting clients ask "can you do POST" etc, and I've become suspicious that maybe `git-annex` even needs this. Or will need it if it ever supports HTTP uploads, where it will need to query "can you do PUT?"